runners/prism/java/src/test/java/org/apache/beam/runners/prism/PrismExecutorTest.java

```diff
@@ -102,4 +102,12 @@ private void sleep(long millis) {
     } catch (InterruptedException ignored) {
     }
   }
+
+  private static String normalizeConsoleOutput(String output) {
```

Comment:
This could be alternately fixed by setting

'--log_kind=json' 
or 
'--log_kind=text'

as a flag to the process here:
 https://github.com/apache/beam/blob/master/runners/prism/java/src/main/java/org/apache/beam/runners/prism/PrismRunner.java#L110

Reply:
Thanks, changed to use '--log_kind=text'